Common/Subworlds/MappingWorld.cs

```diff
@@ -89,8 +98,33 @@ public override void Load()
 		SubworldDescription = Language.GetOrRegister("Mods.PathOfTerraria.Subworlds." + GetType().Name + ".Description", () => GetType().Name);
 		SubworldMining = Language.GetOrRegister("Mods.PathOfTerraria.Subworlds." + GetType().Name + ".Mining", () => "\"{$DefaultMining}\"");
 		SubworldPlacing = Language.GetOrRegister("Mods.PathOfTerraria.Subworlds." + GetType().Name + ".Placing", () => "\"{$DefaultPlacing}\"");
+
+		if (!Main.dedServ)
+		{
+			LoadLoadingScreens();
+		}
 	}
-	
+
+	private void LoadLoadingScreens()
+	{
+		if (ScrollingBackgroundCount == 1 && ModContent.RequestIfExists("PathOfTerraria/Assets/UI/SubworldLoadScreens/" + GetType().Name, out Asset<Texture2D> image))
+		{
+			LoadingBackgrounds = [image];
+		}
+		else if (ScrollingBackgroundCount > 1)
+		{
+			LoadingBackgrounds = new Asset<Texture2D>[ScrollingBackgroundCount];
+
+			for (int i = 0; i < ScrollingBackgroundCount; i++)
+			{
+				if (ModContent.RequestIfExists($"PathOfTerraria/Assets/UI/SubworldLoadScreens/{GetType().Name}_{i}", out Asset<Texture2D> background))
+				{
+					LoadingBackgrounds[i] = background;
+				}
```

Comment:
Will error down the line, so no reason not to throw.
```suggestion
				LoadingBackgrounds[i] = ModContent.Request<Texture2D>($"PathOfTerraria/Assets/UI/SubworldLoadScreens/{GetType().Name}_{i}");
```